numpy/linalg/tests/test_linalg.py

```diff
@@ -729,7 +831,10 @@ def test_square(self):
 
 class TestEigvalsh(HermitianTestCase, HermitianGeneralizedTestCase):
 
-    def do(self, a, b):
+    def do(self, a, b, flags):
+        if flags & CaseFlags.empty:
+            assert_raises(LinAlgError, linalg.eigvalsh, a, 'L')
```

Comment:
So checking for an error is the default (no flags). That is somewhat unintuitive.